tests/integration/nupic/opf/opf_experiment_results_test.py

```diff
@@ -185,7 +185,7 @@ def testExperimentResults(self):
         'results': {
           ('DefaultTask.NontemporalMultiStep.predictionLog.csv',
            "multiStepBestPredictions:multiStep:errorMetric='avg_err':steps=1:window=25:field=field1"):
-                    (0.0, 0.0),
+                    (0.0, 1.0),
```

Comment:
Some ranges in this set of tests had to be readjusted due new SP emmits other values for the same metric.